docs/characterization/parallel_xeb.ipynb

```diff
@@ -0,0 +1,267 @@
+{
```

Comment:
<p>typo: deign</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/quantumlib/Cirq/pull/3760/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>this is a word! I'll change it to something less archaic </p>

Reply:
Sorry, never heard! ;)